src/wp-includes/class-wp-user.php

```diff
@@ -515,6 +534,11 @@ public function get_role_caps() {
 
 		$wp_roles = wp_roles();
 
+		// Edge case: In case someone calls this method before lazy initialization, we need to initialize on demand.
+		if ( ! isset( $this->caps ) ) {
+			$this->caps = $this->get_caps_data();
+		}
```

Comment:
Why wouldn't this also just do?

```suggestion
	$this->load_capability_data();
```

Reply:
This was originally considers. But `load_capability_data` calls `get_role_caps`, which calls `load_capability_data`. This will loop forever.

Reply:
Ah, OK 😄 ♾️